File: caesar_cipher.cpp
// ma Caesar
#include <bits/stdc++.h>
using namespace std;

string caesar(string s, int shift) {
	string result = "";
	for(int i = 0; i < s.length(); i++) {
		if(isupper(s[i])) {
			result += char(int(s[i] + shift - 65) % 26 + 65);
		}
		else {
			result += char(int(s[i] + shift - 97) % 26 + 97);
		}
	}
	return result;
}

int main() {	
	string s;
	int shift;
	getline(cin, s);
	cin >> shift;
	cout << "Sau ma hoa Caesar: " << caesar(s, shift);
}

File: caesar_cipher.py
def caesar(s, shift):
    result = ""
    for char in s:
        if char.isupper():
            result += chr((ord(char) + shift - 65) % 26 + 65)
        else:
            result += chr((ord(char) + shift - 97) % 26 + 97)
    return result

s = input("Nhap chuoi can ma hoa: ")
shift = int(input("Nhap so buoc: "))
print("Chuoi sau khi ma hoa: ", caesar(s, shift))